### PR TITLE
chore(home): Remove farm auctions banner from homepage

### PR DESCRIPTION
**Pull request by ChefHutch**
[ ] Before opening a pull request, please read the [contributing guidelines](https://github.com/pancakeswap/pancake-frontend/blob/master/CONTRIBUTING.md) first
[ ] If your PR is work in progress, open it as `draft`
[ ] Before requesting a review, all the checks need to pass
[ ] Explain what your PR does

### CODE REVIEW

**Review by Chef-Cheems — APPROVED** [review]

**Review by ChefKai — APPROVED** [review]

File: src/views/Home/index.tsx [hunk 1]
```diff
@@ -14,7 +13,6 @@ import Footer from './components/Footer'
 import CakeDataRow from './components/CakeDataRow'
 import { WedgeTopLeft, InnerWedgeWrapper, OuterWedgeWrapper, WedgeTopRight } from './components/WedgeSvgs'
 import UserBanner from './components/UserBanner'
-import FarmAuctionsBanner from './components/FarmAuctionsBanner'
```
Comment (RabbitDoge) [inline src/views/Home/index.tsx:17]: remove the component if it's not used anymore

**Review by RabbitDoge — APPROVED** [review]
remove dead components before merging plz

File: src/views/Home/index.tsx (hunk 1 above)
Comment (ChefHutch) [reply src/views/Home/index.tsx:17]: Going to be used again in a week 👍

Comment (ChefHutch) [reply src/views/Home/index.tsx:17]: I've deleted and then re-added banners so many times. Plz spare this lil guy

**Pull request merged**